metadata/src/main/java/org/apache/kafka/image/loader/LogDeltaManifest.java

```diff
@@ -66,6 +119,10 @@ public LogDeltaManifest(
         this.numBytes = numBytes;
     }
 
+    public static Builder newBuilder() {
```

Comment:
curious why this is better than just having a public Builder constructor (I don't feel strongly, I guess...)

Reply:
Well, it also has a public no-arg constructor, I just like the static factory. I do like the fluent style for builders and a static method to start it off makes sense to me.

Reply:
ok